File: codekata/basics/scalene_triangle.py
# scalene triangle
# get three values as input from user in a single line
A,B,C=list(map(int,input().split()))
# define scalene function
def scalene():
# if condition check whether the given values are scalene triangle or not
  if(A!=B and B!=C and C!=A):
    c="yes"
    return c
  else:
    c="no"
    return c
# call scalene function and store it in result
result=scalene()
# print result value
print(result)

File: codekata/javascript/basics/scalene_triangle.js
// print 'yes' if they can form the sides of a scalene triangle else print 'no'
const readline = require('readline');
const inp = readline.createInterface({
  input: process.stdin
});
const userInput = [];
inp.on("line", (data) => {
	userInput.push(data);
});
inp.on("close", () => {
    var d=userInput[0].split(" ");
    var a=d[0],b=d[1],c=d[2];
	if(a!=b && b!=c && a!=c)
      console.log("yes");
  else
    console.log("no");
});
